CRM/Core/BAO/CustomGroup.php

```diff
@@ -74,7 +74,12 @@ public static function create(&$params) {
       $extendsChildType = $params['extends_entity_column_value'];
     }
     if (!CRM_Utils_System::isNull($extendsChildType)) {
-      $extendsChildType = implode(CRM_Core_DAO::VALUE_SEPARATOR, $extendsChildType);
+      if (is_array($extendsChildType)) {
```

Comment:
I dug into this & it's a pre-existing nastiness / probable bug.

The values being passed in are

```'extends' => ['Individual', 'Household']```

which seems pretty OK / valid 

However, the code, a few lines above - is trying to helpfully interpret 'Household' as a child type - which probably reflects how a particular form works.

Probably we should fix the form to pass in 

'extends' as a string or an array as supported elsewhere and
'extends_entity_column_value' as a string or array too and not try to do unexpected stuff with what is in extends

Note that I think merging this as is would cause a bug since it is totally legit for and even expected for extendsChildType to be a string at this point

Reply:
Yeh I'm not sure what is correct but it seems to have been relying on the fact that implode doesn't like non arrays as the 2nd parameter.  Perhaps it is just a bad test but not 100% sure here